pkg/gameserverallocations/allocator.go

```diff
@@ -82,6 +82,13 @@ const (
 	batchWaitTime         = 500 * time.Millisecond
 )
 
+const (
+	// Timeout for an individual Allocate RPC call can take
+	remoteAllocateTimeout = 10 * time.Second
```

Comment:
@kdima - is it worth making these CLI args of the allocator? In an ideal world, I think we'd want to set these to like 1s

Reply:
Yeah it's a bit weird. I could not find a good place to add them. Will ask the maintainers